webapp/src/css/inbox.less

```diff
@@ -93,178 +93,108 @@ body {
-
-    .open > a {
-      background-color: @white;
-
-      .mm-icon {
-        color: @top-header-color;
-      }
-    }
-  }
-
-  .dropdown ul a {
-    width: 100%;
-    height: auto;
-  }
-
-  .dropdown.options.open.show {
-    display: inline !important; // Overriding BS' "display" that uses !important;
+    display: none;
   }
 
-  .options .dropdown-menu {
-    width: @header-dropdown-width;
-    top: 0 !important; // Overriding element.style
-    left: calc(-1 * (@header-dropdown-width - @header-bar-extras)) !important; // Overriding element.style
-    margin-top: 30px;
-    font-size: 0;
-    a {
-      text-shadow: none;
-      padding: 10px 20px;
-      color: @gray-ultra-darker;
-      margin-top: 0;
-      white-space: normal;
-      font-size: @font-medium;
-    }
-    .disabled a {
-      color: @gray-disabled;
-    }
-    .divider {
-      margin: 0;
-    }
-    .sync-status {
-      margin-top: 10px;
-      margin-bottom: 10px;
-      a {
-        padding-top: 0;
-        padding-bottom: 0;
-        color: @label-color;
-        &:hover {
-          background-color: transparent;
-          cursor: default;
-        }
-      }
-      .success {
-        color: @ok-color;
-      }
-      .required {
-        color: @error-color;
-      }
-    }
+  .inner {
+    height: 100vh;
   }
 
   .tabs {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    height: 100%;
+    padding: 0 15px;
     overflow: hidden;
     white-space: nowrap;
 
-    .button-label {
-      top: -5px;
-      position: relative;
-      overflow: hidden;
-      word-wrap: break-word;
-      color: @nav-icon-gray;
-    }
-
     a {
+      .button-text-inverse(@nav-icon-gray);
+      outline: 0;
+      text-shadow: none;
       border-radius: @radius-size @radius-size 0 0;
+      padding: 0 5px;
+      margin-top: 30px;
       cursor: pointer;
       min-width: 80px;
-      padding-left: 5px;
-      padding-right: 5px;
 
-      .mm-icon {
-        display: block;
-        padding: 5px 0;
-        height: 34px;
-        width: 54px;
-
-        svg, span:not(.mm-badge-overlay):not(.mm-badge), img {
-          &, &:before {
-            display: block;
-            margin: auto;
-            height: 20px;
-            max-width: 20px;
-            overflow: hidden;
-          }
-        }
+      &:hover,
+      &:focus {
+        text-decoration: none;
       }
 
       &.selected {
         text-shadow: none;
+        .mm-icon {
+          border-radius: 999px;
+          background-color: @tab-selected-color;
+        }
+
         .button-label,
         .mm-icon:hover,
         .resource-icon {
           .button-text(@white);
           text-shadow: none;
         }
       }
+    }
 
-      &:hover,
-      &:focus {
-        text-decoration: none;
+    a .button-label {
+      margin-top: 3px;
+      position: relative;
+      overflow: hidden;
+      word-wrap: break-word;
+      color: @nav-icon-gray;
+      font-size: @font-XXS;
```

Comment:
Should we add ellipsis so that when the work label is too long there's a clear indication that the word has been cut ?